package.json

```diff
@@ -97,6 +98,7 @@
     "mailparser": "0.6.1",
     "mkdirp": "0.5.1",
     "mocha": "3.1.2",
+    "mustache": "2.3.0",
```

Comment:
This was done in 1ed302824b80f0399b935aa98f6a686a94b83b79 although fwiw, and purely as an aside, I find the mustache syntax objectively nicer.

No need for `../` to pick up the parent scope, no weird `{{#if}}`/`{{#with}}` duality and specific terminating tags are more helpful to read than generic `{{/each}}`, `{{/if}}` etc. (imho)